ios-base/DroidKaigi 2020/ApplicationScheme.swift

```diff
@@ -36,7 +36,7 @@ final class ApplicationScheme: NSObject {
         scheme.primaryColor = UIColor(hex: "041E42")
         scheme.onPrimaryColor = .white
         scheme.secondaryColor = UIColor(hex: "041E42")
-        scheme.surfaceColor = .white
+        scheme.surfaceColor = UIColor(hex: "041E42")
```

Comment:
It looks like the sidebar doesn't use `surfaceColor`.

I want to use `surfaceColor` as `.white` on [another screen](https://github.com/DroidKaigi/conference-app-2020/pull/585), so can it be left `.white`? 🙏

Reply:
Oh, I'm sorry. I will fix it.